.travis.yml

```diff
@@ -114,12 +175,18 @@ script:
       tox -v
     fi
     if [[ -n ${WHEEL_PATH:-} ]]; then
+      if [[ ${TRAVIS_CPU_ARCH} == "arm64" ]]; then
+         sudo apt-get install patchelf
+         python3 -mpip install --progress-bar=off auditwheel
+         python3 -m auditwheel repair $WHEEL_PATH/*.whl -w $WHEEL_PATH/
+         rm -rf $WHEEL_PATH/*linux_aarch64.whl
+      fi
       twine upload --repository-url https://test.pypi.org/legacy/ --skip-existing $WHEEL_PATH/*.whl
     fi
     )
 after_failure:
-  - more .tox/log/* | cat
-  - more .tox/*/log/* | cat
+  - travis_wait 15 more .tox/log/* | cat
```

Comment:
What's the point of using the waiter here?

Reply:
When the build fails in arm64, this command was giving timeout error when tested locally. After adding this code, command is executing properly.

Reply:
To be honest now I wonder why I didn't have just `cat  .tox/log/*` directly in there.

Reply:
I have updated that command to `cat .tox/log/*`.